gtsfm/utils/io.py

```diff
@@ -297,8 +303,10 @@ def write_images(gtsfm_data: GtsfmData, images: List[Image], save_dir: str) -> N
             iTw = camera.pose().inverse()
             iRw_quaternion = iTw.rotation().quaternion()
             itw = iTw.translation()
-            tx, ty, tz = itw
-            qw, qx, qy, qz = iRw_quaternion
+
+            iuprightRi, iupright_t_i = ellipsoid_utils.transform_camera_frustums(itw, iRw_quaternion, mean, wuprightRw)
```

Comment:
thanks for the PR, @adisingh50.

Let's pull the orthogonal alignment out of the file I/O, and put it into the scene optimizer itself. So that the COLMAP writing function here is purely I/O, no computation.

Could you do that? Could you pull in master to this branch, also?